config.example/group_vars/all.yml

```diff
@@ -96,3 +96,29 @@ users:
     groups: ['sudo']
     uid: 10007
     home: /home/nvidia
+
+# Zabbix
+# Web User: Admin, Pass: zabbix
+postgresql_users:
+  - name: zabbix-server
+    password: zabbix-server
```

Comment:
This defaults to creating `zabbix-server` as an unprivileged user, so it can't create the Zabbix DB when it needs to. In order to finish the Ansible run, I manually added privileges to the user via `psql`.

You probably want this to look like:

```
postgresql_users:
    - name: zabbix-server
      password: zabbix-server
      priv: 'ALL'
```

But I haven't tested that. :) Docs here: https://docs.ansible.com/ansible/latest/modules/postgresql_user_module.html